bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amp-viewer",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
@frankiefu is it fine to increment the version in this PR? or you'd prefer to do it in a separate PR?

Reply:
I think this is fine.